site_scons/prereq_tools/base.py

```diff
@@ -443,7 +443,9 @@ def __init__(self, env, opts):
         opts.Add('USE_INSTALLED', 'Comma separated list of preinstalled dependencies', 'none')
         opts.Add(('MPI_PKG', 'Specifies name of pkg-config to load for MPI', None))
         opts.Add(BoolVariable('FIRMWARE_MGMT', 'Build in device firmware management.', 0))
-        opts.Add(BoolVariable('STACK_MMAP', 'Allocate ABT ULTs stacks with mmap()', 0))
+        opts.Add(BoolVariable('STACK_MMAP', 'Allocate ABT ULTs stacks with mmap()', 1))
+        opts.Add(BoolVariable('ADDRESS_SANITIZER',
+                              'Instrument C code with the Address Sanitizer library', 1))
```

Comment:
Would `SANITIZER=<address|thread|...>` be more economical? Or, even `COMPILER_FEAT=<asan|tsan|...>`, but I'm not sure if that's going too far.